.rubocop.yml

```diff
@@ -18,6 +18,12 @@ AllCops:
     - 'truffleruby/target/classes/**/*'
     - 'mxbuild/**/*'
 
+Lint/LiteralInCondition:
+  Description: Checks of literals used in conditions.
+  Enabled: true
+  Exclude:
+  - lib/truffle/java/java_ext/*.rb
```

Comment:
I agree on enabling this, after looking at the current cases it flags in core, it seems like those could use some improvement anyway.